tests/manage/z_cluster/test_ceph_default_values_check.py

```diff
@@ -109,3 +116,31 @@ def test_validate_ceph_config_values_in_rook_config_override(self):
             f"The expected values are:\n{stored_values}\n"
             f"The cluster's Ceph values are:{config_data}"
         )
+
+    @post_ocs_upgrade
+    @skipif_ocs_version("<4.7")
```

Comment:
Why we are skipping for releases less than 4.7, I think the value was changed to 4GB long time back